proposals/description/fip_82b.ts

```diff
@@ -406,6 +416,26 @@ const fip_82b: ProposalDescription = {
         '{optimisticTimelock}'
       ],
       description: 'Grant optimisticTimelock the TOKEMAK_DEPOSIT_ADMIN_ROLE role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x6c9ecf07a5886fd74a8d32f4d3c317a7d5e5b5c7a073a3ab06c217e9ce5288e3', // PCV_SAFE_MOVER_ROLE
+        '{tribalCouncilTimelock}'
+      ],
+      description: 'Grant tribalCouncilTimelock the PCV_SAFE_MOVER_ROLE role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantRole(bytes32,address)',
+      arguments: [
+        '0x6c9ecf07a5886fd74a8d32f4d3c317a7d5e5b5c7a073a3ab06c217e9ce5288e3', // PCV_SAFE_MOVER_ROLE
+        '{optimisticTimelock}'
```

Comment:
Backwards compatibility, TribalCouncil can later revoke when we deprecate the old OA system

Reply:
Does optimistic timelock have Guardian or PCV_ADMIN? If not I prefer removing this

Reply:
Also, we'd need to actually deploy a new PCVGuardian and transfer the roles over

Reply:
Removing and deploying a new PCVGuardian with migrated state